homeassistant/components/tolo/number.py

```diff
@@ -17,113 +24,84 @@ async def async_setup_entry(
     coordinator = hass.data[DOMAIN][entry.entry_id]
     async_add_entities(
         [
-            ToloPowerTimerDuration(coordinator, entry),
-            ToloSaltBathTimerDuration(coordinator, entry),
-            ToloFanTimerDuration(coordinator, entry),
+            ToloNumberEntity(coordinator, entry, description)
+            for description in (
+                ToloNumberEntityDescription(
```

Comment:
You are getting here! Nice!

Maybe move this tuple into a constant

Reply:
If I make it a top level constant, `coordinator` is not defined anymore and I would have to introduce lambda expressions, which IMHO is much harder readable as the current version.

I could also make it a local constant in `async_setup_entry` method, which (again, IMHO) does not really increase readability compared with current version.

Which way would you prefer?

Reply:
We currently prefer lambda's (search `value_fn` for examples)

Reply:
Done.